### DISCUSSION TITLE
Does TCLB support Large eddy simulation of turbulence?

### DISCUSSION THREAD

**Discussion by ztdepztdep** [CFD-GO/TCLB#350] [Q&A] [answered]
Does TCLB support Large eddy simulation of turbulence?

**Comment by llaniewski** [comment, marked as answer]
Some models include the [Smagorinsky model](https://en.wikipedia.org/wiki/Large_eddy_simulation#Smagorinsky%E2%80%93Lilly_model) for LES. For example the `d3q27_cumulant` model. You can activate it by setting the Smagorinsky constant `Smag` to anything other than 0.
